Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8192,3 +8192,14 @@ def get_tenant_account_name():
         account_name = "acc_{}".format(tenant_name) if tenant_name != "" else ""
 
     return account_name
+
+def indicator_value_to_clickable(indicator):
+    res = demisto.executeCommand('GetIndicatorsByQuery', {'query': f'value:{indicator}'})
+    if isError(res[0]):
+        raise DemistoException('Query for get indicators is invalid')
```

Comment:
Let's use `IndicatorsSearcher` class instead. If you're looking for just one value you can use it like so:
```
            search_indicators = IndicatorsSearcher()
            data = search_indicators.search_indicators_by_version(value=indicator).get('iocs')
```
But better yet, provide an option to run it with a list of indicators.